### ISSUE TITLE
Updating the CONTRIBUTING.md file

### ISSUE THREAD

**Issue by matthewsot** [UC-Davis-CS-Tutoring/Athena#2]
I plan to update the contributing.md file a bit to fit the scale (and name) of the project. I'm hoping to have one single CONTRIBUTING.md in this repository that can then just be referenced by links in all of the other UC-Davis-CS-Tutoring repositories.

**Comment by tehalexf** [comment]
Thank you for doing this! 

I think it is a good idea to have a central contributing.md, but this repo might be the wrong place for it, as Athena is a tutoring suite I've been developing solo for a while now. 

Mainly, on line 9 of contributing.md, I'd like you to be the point of contact for every other repo, but for Athena only, the default point of contact should be my email, aafu@ucdavis.edu.

Another alternative would be for me to move Athena out of the UCDCST repo altogether. 

This brings up another issue I had a while ago -- I didn't intend for this repo itself to be contributed to necessarily. Yes, it's open source, but that is to demonstrate the ability to host everything on a server-less environment and to make the source available to those who want to see it. As for others contributing to it however, I have unrealistic standards that hypocritically do not apply to myself. I would reconsider this approach once the project is stable and has sufficient documentation.

Please let me know what you think.

Alex

**Comment by tehalexf** [comment]
Eh, I think I took the nuclear approach way too fast, apologies.

I would be fine hosting a generic contributing.md if it specifically called out Athena as having myself as a Point of Contact to discuss changes.

Other than that, I agree with the idea to merge all the contributing.mds.

Alex

**Comment by matthewsot** [comment]
Hi Alex, thanks for the reply!

I think it would actually be ideal to have a separate repository for "logistical" things like this, but I obviously don't want to clutter the organization up with too many repositories. I think I'll start a new UCDCST/Common repository and put this there (as long as hopefully a few other things that might be useful to throw there).

With respect to the future of this code within UCDCST, I've been talking to @aakash1104 a bit about that, and I think we should discuss more over chat/in-person! (Which Aakash will set up). There are further plans to extend this to a more feature-rich project, and it may be easier to take a different approach than extending this repo for those new features.

So, to sum up -- I'll just start a separate repo for this and link to that from all the other repositories.

**Comment by matthewsot** [comment]
Actually, re-opening: if you're interested in standardizing, I just pushed up a CONTRIBUTING.md that references the Common one and lists your email as the point of contact.

**Comment by matthewsot** [comment]
https://github.com/UC-Davis-CS-Tutoring/Athena/blob/universal-contributing-file/CONTRIBUTING.md

Feel free to pull that into master or make changes or leave master as is and close this issue!

**Comment by tehalexf** [comment]
This is perfect :)

I'm an alumni that lives on the east coast now, and Aakash told me you were interested in taking a bigger part in this project as a whole, as well as inheriting, being the new tutoring head and all :). I want to let you know that I'm all for it, but will need a month or so to tune Athena a bit, after which it should be fine to accept other contributions (scaffolding linting, unit tests, etc will take approximately this much time).

I look forward to talking with you soon!

Alex

**Comment by tehalexf** [comment]
Merged branch, closing issue :)
